modules/distribution/src/repository/resources/dbscripts/apimgt/mssql.sql

```diff
@@ -259,11 +261,11 @@ IF NOT  EXISTS (SELECT * FROM SYS.OBJECTS WHERE OBJECT_ID = OBJECT_ID(N'[DBO].[S
 CREATE TABLE SP_INBOUND_AUTH (
             ID INTEGER NOT NULL IDENTITY,
             TENANT_ID INTEGER NOT NULL,
-            INBOUND_AUTH_KEY VARCHAR (255) NOT NULL,
+            INBOUND_AUTH_KEY VARCHAR (255),
             INBOUND_AUTH_TYPE VARCHAR (255) NOT NULL,
             PROP_NAME VARCHAR (255),
             PROP_VALUE VARCHAR (1024) ,
-            APP_ID INTEGER NOT NULL,
+       	    APP_ID INTEGER NOT NULL,
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04